components/director/cmd/director/main.go

```diff
@@ -86,8 +91,9 @@ import (
 const envPrefix = "APP"
 
 type config struct {
-	Address string `envconfig:"default=127.0.0.1:3000"`
-	AppURL  string `envconfig:"APP_URL"`
+	Address         string `envconfig:"default=127.0.0.1:3000"`
+	InternalAddress string `envconfig:"default=127.0.0.1:3002"`
```

Comment:
The `InternalAddress`, just like the `Address` should be defined somewhere centrally in our charts and configurable through environment variables. This way the host+port will be defined in one place and all other components which needed, like the `operations-controller` and charts, like the network policy, can reference it.